backend/ee/danswer/danswerbot/slack/handlers/handle_standard_answers.py

```diff
@@ -21,11 +21,11 @@
 from danswer.db.models import Prompt
 from danswer.db.models import SlackBotConfig
 from danswer.db.models import StandardAnswer as StandardAnswerModel
```

Comment:
Often we'll call it DbStandardAnswer and the other one will be just StandardAnswer for the server model

Reply:
`StandardAnswerModel` is the original naming from this file, do you want me to change it as part of this refactor?

Reply:
Doesn't matter, nobody will get confused by this anyway